packages/components/src/styles/components/side-nav/header.scss

```diff
@@ -90,19 +57,24 @@
   gap: var(--token-side-nav-header-actions-spacing);
 }
 
-// dropdown overrides
+// Dropdown & Button color theming overrides
 
-.hds-side-nav__dropdown { // ⬅︎ add this class name to the `Hds::Dropdown` component to apply the style overrides
-  .hds-dropdown-toggle-icon,
-  .hds-dropdown-toggle-button {
-    @include hds-side-nav-icon-button($add-visible-border: true);
+.hds-side-nav-header,
+.hds-side-nav__wrapper-body,
+.hds-side-nav__wrapper-footer {
+  // Apply dark theme to child interactive components
+  .hds-button,
+  .hds-dropdown-toggle-button,
+  .hds-dropdown-toggle-icon {
+    @include hds-interactive-dark-theme();
```

Comment:
Previously the interactive-dark-theme colors were applied to SideNav Dropdowns using a classname, ".hds-side-nav__dropdown". This change removes the need for a classname to instead rely upon inheritance as I did for Buttons & Dropdowns in the AppHeader.

I did a search for the classname within consumer codebases and it was not being overridden anywhere: https://github.com/search?q=org%3Ahashicorp+hds-side-nav__dropdown&type=code

However, if you think it would be better to instead deprecate the class name and remove in a separate PR I can do so.

Reply:
@KristinLBradley what I can see from the search results is that it's not being _overwritten_, but it's being _used_ to apply a dark mode to some dropdowns: https://github.com/search?q=org%3Ahashicorp+hds-side-nav__dropdown+-repo%3Ahashicorp%2Fdesign-system&type=code

From what I understand
- [here](https://github.com/hashicorp/consul/blob/main/ui/packages/consul-ui/app/components/hashicorp-consul/index.hbs#L88) it's used in the SideNav header, so you should add `hds-side-nav__wrapper-header` to the selector above
- [here](https://github.com/hashicorp/consul/blob/main/ui/packages/consul-acls/app/components/consul/token/selector/index.hbs#L60) + [here](https://github.com/hashicorp/consul/blob/main/ui/packages/consul-ui/app/components/hashicorp-consul/index.hbs#L108) it's being used also in the SideNav header
- other instances too seem to also in the context of the SideNav
So _in theory_ it should be ok to just remove the class and rely on inheritance. One consideration is that, since the `hds-side-nav__wrapper-[header/body/footer]` are practically the only containers for the `SideNav` (apart from the `root/yield`) then maybe you can simplify things and just use `hds-side-nav` as parent class for the inheritance.

Reply:
Thanks, I'll simplify the selectors and also add back the ".hds-side-nav__dropdown" classname for now.

Reply:
Resolving for now, thanks for your help